components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1379,7 +1379,7 @@ union TxDataUnion {
 
 struct TransactionInfo {
   string id;
-  string from_address;
+  string? from_address; // Filled with account address for ETH, SOL and FIL.
```

Comment:
When will this be null?

Reply:
Will bit null for BTC transactions. That PR is in progress.
So each transaction has a 'from Account'(via from_account_id), but not a designated from_address. 
Actually for BTC that should be an array of addresses owned by account. Maybe eventually from_address (and BTC's from_addresses) should move into coin specific parts of TxDataUnion, but I'm not sure now.